apps/apps_entry/Kconfig

```diff
@@ -0,0 +1,24 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+if INIT_ENTRYPOINT
+choice
+	prompt "Application entry point"
+config ENTRY_MANUAL
+	bool "Set entry point manually"
+	---help---
+		If you select this, you should add a function name as an application entry point
+		at "Entry point function name".
+
+comment "Application entry point list"
+source "$APPSDIR/Kconfig_ENTRY"
```

Comment:
It looks not good. You already made apps_entry folder but you don't place Kconfig_ENTRY in there. You had better modify mkkconfig.sh and bat to place that file in apps_entry folder. or remove apps_entry folder.